.github/workflows/build-deploy.yml

```diff
@@ -40,6 +40,12 @@ jobs:
           name: production-files
           path: ./build
 
+      - name: Run ESLint
+        run: yarn run lint . --ext .js,.jsx,.ts,.tsx
+
+      - name: Run prettier
+        run: yarn run prettier --write "./src/**/*.{md,css,json}"
```

Comment:
Please refer back to my earlier [comment](https://github.com/unicef/kindly-website/pull/10#pullrequestreview-804961358): there is no point in running `prettier` because all and any changes that it makes will not be committed, and lost.